WordPress/Classes/System/WPGUIConstants.m

```diff
@@ -1,11 +1,15 @@
 #import "WPGUIConstants.h"
 
-const CGFloat WPAlphaFull = 1.0f;
-const CGFloat WPAlphaZero = 0.0f;
+const CGFloat WPAlphaFull                   = 1.0f;
+const CGFloat WPAlphaZero                   = 0.0f;
 
-const CGFloat WPColorFull = 1.0f;
-const CGFloat WPColorZero = 0.0f;
+const CGFloat WPColorFull                   = 1.0f;
+const CGFloat WPColorZero                   = 0.0f;
 
-const CGFloat WPAnimationDurationDefault = 0.33f;
-const CGFloat WPAnimationDurationFast = 0.15f;
-const CGFloat WPAnimationDurationFaster = 0.07f;
\ No newline at end of file
+const CGFloat WPAnimationDurationDefault    = 0.33f;
+const CGFloat WPAnimationDurationFast       = 0.15f;
+const CGFloat WPAnimationDurationFaster     = 0.07f;
+
+const CGFloat WPTableViewTopMargin          = 40.0f;
+const CGRect WPTableHeaderPadFrame          = {0.0f, 0.0f, 0.0f, 40.0f};
+const CGRect WPTableFooterPadFrame          = {0.0f, 0.0f, 0.0f, 48.0f};
```

Comment:
Nuked the `f`'s. Regarding the WPStyleGuide category, i've got mixed feelings about it, since we're not applying the exact same fix everywhere.

I'm thinking we should, perhaps, review WPTableViewCell usage, along with this iPad Header + Footer frame, and get it all right in one shot!